handbook/engineering/web/frontend-platform/wildcard-component-library.md

```diff
@@ -0,0 +1,53 @@
+# Wildcard Component Library (WCL)
+
+Wildcard Component Library is a comprehensive collection of reusable, design-approved components that can be composed to quickly build out different user journeys across the web app. WCL is stored in its own [directory](https://github.com/sourcegraph/sourcegraph/tree/main/client/wildcard), owned by the [Fronted Platform Team](https://about.sourcegraph.com/handbook/engineering/web/frontend-platform) and contributed to by all Sourcegraph teammates. WCL is a part of the Wildcard Design System.
+
+## WCL x Frontend Platform Team
+- The Frontend Platform Team is the owner of the design system and Wildcard Component Library  and needs to be up-to-date with all the changes and needs of other teams
+- The Frontend Platform Team should enable other teams to make changes to the Wildcard Component Library by keeping them up-to-date (see Sharing updates to the components system), helping them plan their work, providing expertise, and reviewing the changes
+- The Frontend Platform team **shouldn't be responsible for implementing components for other teams**. Otherwise, we would create a bottleneck and FE Platform team wouldn’t have time to focus on their team goals.
```

Comment:
```suggestion
- The Frontend Platform team **shouldn't be exclusively responsible for implementing components for other teams**. Otherwise, we would create a bottleneck and the FE Platform team wouldn’t have time to focus on their team goals.
```

I think it is OK for the team to sometimes build components for other teams if it aligns with our roadmap or it introduces a new behavior/pattern that would be better if it was started by the FPT.

Reply:
I agree with @jeanduplessis's suggestion. The frontend platform team will allocate time for adding components and documentation to the Wildcard library, but it should not be solely responsible for doing that. 

> This is a serious risk too for our design/UI quality because this means over time the UI becomes inconsistent

Improving the Wildcard library docs and processes around introducing new components should help with that. FTP should facility best practices and make product teams aware of them. Another direction that we should look at is making our linters sticker to ensure that there's one correct way of doing things. Consistency shouldn't require the full attention of some particular team, but rather be the result of the processes and practices established for the front-end development.